src/NovelRT/Audio/AudioService.cpp

```diff
@@ -2,6 +2,7 @@
 // for more information.
 
 #include <NovelRT/Audio/Audio.h>
+#include <sndfile.h>
```

Comment:
Is there a reason we moved it into here?

Reply:
I had moved it here so anything referring to the Audio header wasn't also pulling in everything related to sndfile

Reply:
Encapsulating dependencies is usually a good thing. I approve.